steps/hive.py

```diff
@@ -0,0 +1,59 @@
+from logging import getLogger
+
+logger = getLogger("hive")
+
+
+class HiveService:
+    def __init__(
+            self,
+            transition_db_name: str,
+            db_name: str,
+            correlation_id: str,
+            spark_session,
+    ):
+        self._transition_db_name = transition_db_name
+        self._db_name = db_name
+        self._correlation_id = correlation_id
+        self._spark_session = spark_session
+
+    def execute_queries(self, queries):
+        for query in queries:
+            if query and not query.isspace():
+                logger.info(
+                    f"Executing query : {query} for correlation id : {self._correlation_id}",
+                    query,
+                    self._correlation_id,
+                )
+                self._spark_session.sql(query)
+            else:
+                logger.info(f"Not executing invalid query : {query} for correlation id : {self._correlation_id}")
```

Comment:
L29 generates a slightly confusing log - because a query of " " looks like this:

Not executing invalid query : &nbsp; &nbsp; for correlation id <...>
-----------------------------^ there is no query